doc/man/cockpit.conf.xml

```diff
@@ -108,7 +108,7 @@ ForwardedForHeader = X-Forwarded-For
             on the login screen is visible and allows logging into another server. When set to
             <literal>false</literal>, direct remote logins are disallowed. If this option is not specified
             then it will be automatically detected based on whether the
-            <command>cockpit-ssh</command> process is available or not.</para>
+            <command>cockpit-bridge</command> and <command>ssh</command> programs are available.</para>
```

Comment:
Why check for cockpit-bridge? Does its present somehow mean that `python -m cockpit.beiboot` will work? That would be a bit indirect, no?

Reply:
Right, this is an optimization. `python3 -m cockpit.beiboot` is awfully expensive:

```
❱❱ time python3 -m cockpit.beiboot --help
real	0m0.408s
```

This is the *second* call, so hot cache. `--help` so that it exits with zero in the success case (without arguments it'd fail on invalid options).

OTOH `time command -v cockpit-bridge` is unmeasurable (0m0.000s).

Reply:
Right, makes sense. But does the documentation need to expose this? It could still talk about "the cockpit.beiboot Python module" being needed, and the code uses the knowledge that that module is right now in the cockpit-bridge package, which we detect by looking whether the cockpit-bridge command is in $PATH.

Reply:
Ah, you literally meant the documentation -- I thought this was a proxy place for the actual implementation.

I changed it to "the cockpit-bridge package is installed", which is I think the best concept for users -- `cockpit.beiboot` Python module is too much of an implementation detail  for my taste, it may change (*cough* renaming), and it's not user-visible API or CLI.

Reply:
> Ah, you literally meant the documentation -- I thought this was a proxy place for the actual implementation.

Well, I guess I changed my mind without noticing. I didn't start out with a clear plan here, tbh. :-)

Reply:
> I changed it to "the cockpit-bridge package is installed", which is I think the best concept for users

Ack.